From 90156575f833d90b984c862d541ac9d05324e4ca Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Thu, 08 Apr 2021 13:29:23 +0300
Subject: [PATCH] [SW-0] preboot: move weak function

Move weak preboot_finalize_fw_info() to fit.c as this a more proper
place for it since this module that uses it.

No functional changes.

Change-Id: Ibb4cd14e16eed4421632377341b202d7d2c54258
Signed-off-by: Guy Nisan <gnisan@habana.ai>
---

diff --git a/preboot/fit.c b/preboot/fit.c
index 5d9af76..580623a 100644
--- a/preboot/fit.c
+++ b/preboot/fit.c
@@ -849,6 +849,8 @@
 	return 1;
 }
 
+__weak void preboot_finalize_fw_info(void) {}
+
 void fit_run_image(void)
 {
 	u64 startaddr = 0;
diff --git a/preboot/preboot.c b/preboot/preboot.c
index 08ee3b4..6c53dd8 100644
--- a/preboot/preboot.c
+++ b/preboot/preboot.c
@@ -267,8 +267,6 @@
 	return false;
 }
 
-__weak void preboot_finalize_fw_info(void) {}
-
 void preboot_boot_info_set_fw_info(void)
 {
 	fw_info.version = HL_FW_INFO_VERSION;
